searchlib/src/vespa/searchlib/attribute/imported_attribute_vector_read_guard.h

```diff
@@ -41,7 +41,7 @@ class ImportedAttributeVectorReadGuard : public IAttributeVector,
 
 protected:
     uint32_t getTargetLid(uint32_t lid) const {
-        return _targetLids[lid];
+        return lid < _targetLids.size() ? _targetLids[lid] : 0u;
```

Comment:
Consider making a unit test for this. E.g in imported_attribute_vector_test.cpp. Would also be nice with a comment on why this is needed.